<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="answer2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KCHEC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KCHEC</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_lal">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="timer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KCHEC</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CHECK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answer1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KCHEC</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
